packages/web-service/src/handlers/__tests__/error-handler.spec.js

```diff
@@ -29,7 +29,7 @@ describe('the error-page handler function', () => {
     }
     await errorHandler(request, h)
     expect(mockView).toHaveBeenCalledWith('client-error', expect.any(Object))
-    expect(mockCode).toHaveBeenCalledWith(400)
+    expect(mockCode).toHaveBeenCalledWith(404)
   })
 
   it('it returns the server-error page with a error status 500', async () => {
```

Comment:
I think now its best not to refer to these two pages as the client and server pages.

Basically we have the ERROR page and the NOT-FOUND page

404 is mapped to the NOT-FOUND page

All other 4XX and 5XX are mapped to the error-page

I can't see an explicit test for the 4XX !404